src/gemm.jl

```diff
@@ -95,31 +93,27 @@ for (gemm, elt) in gemm_datatype_mappings
             # In some tests, size (20,20,20) is worth splitting between two threads,
             # as is size (32,32,8).
 
-            if n_threads > 1
-
-                old_threads = get_num_threads()
-                set_num_threads(1)
-                Threads.@sync for ks in Iterators.partition(1:size(C, 3), cld(size(C, 3), n_threads))
-                    Threads.@spawn for k in ks
-
-                        ptrAk = ptrA + (k-1) * strA * sizeof($elt)
-                        ptrBk = ptrB + (k-1) * strB * sizeof($elt)
-                        ptrCk = ptrC + (k-1) * strC * sizeof($elt)
-
-                        ccall((@blasfunc($(gemm)), libblas), Nothing,
-                              (Ref{UInt8}, Ref{UInt8}, Ref{BlasInt}, Ref{BlasInt},
-                               Ref{BlasInt}, Ref{$elt}, Ptr{$elt}, Ref{BlasInt},
-                               Ptr{$elt}, Ref{BlasInt}, Ref{$elt}, Ptr{$elt},
-                               Ref{BlasInt}),
-                              transA, transB, m, n,
-                              ka, alpha, ptrAk, max(1,Base.stride(A,2)),
-                              ptrBk, max(1,Base.stride(B,2)), beta, ptrCk,
-                              max(1,Base.stride(C,2)))
-                    end
-                end
-                set_num_threads(old_threads)
-
-            else # small problem, no threads
+            # TODO fix multithreading problems
```

Comment:
What's the reason for this?

Reply:
it was producing wrong results in the multithreaded CI environment. Didn't investigated the causes

Reply:
Got a link? I can't reproduce locally.

Reply:
can't find it. You could comment in again this branch and file a PR to trigger CI

Reply:
The commit "fix compat" comments these out. But at that point, `using Compat` has already been deleted, so it will fail, not with the wrong results, but because the function from Compat.jl is not being imported. 

That was deleted in "fix tests and cleanup", the only change to this file in that commit.